### ISSUE TITLE
Can't spawn harming potion item

### ISSUE THREAD

**Issue by pop4959** [EssentialsX/Essentials#2349]
### Information

**Full output of `/ess version`**:

```
[03:20:32 INFO]: Server version: 1.13.2-R0.1-SNAPSHOT git-Paper-492 (MC: 1.13.2)
[03:20:32 INFO]: EssentialsX version: 2.16.0.3
[03:20:32 INFO]: PermissionsEx version: 1.23.4
[03:20:32 INFO]: Vault version: 1.5.6-b49
[03:20:32 INFO]: EssentialsXProtect version: 2.16.0.3
[03:20:32 INFO]: EssentialsXGeoIP version: 2.16.0.3
[03:20:32 INFO]: EssentialsXChat version: 2.16.0.3
[03:20:32 INFO]: EssentialsXAntiBuild version: 2.16.0.3
[03:20:32 INFO]: EssentialsXSpawn version: 2.16.0.3
[03:20:32 INFO]: You are running unsupported plugins!
```

**Server log**: N/A

**EssentialsX config**: N/A

### Details

**Description**  
Can't spawn harming potion item

**Steps to reproduce**  
Type the command in the screenshot

**Expected behavior**  
Spawns a harming potion

**Screenshots**  
![https://i.gyazo.com/3e1559407662b7c193190cdc4d4db7f8.png](https://i.gyazo.com/3e1559407662b7c193190cdc4d4db7f8.png)

![https://i.gyazo.com/4f08e8eba7ceb1ceb836286e12610a02.png](https://i.gyazo.com/4f08e8eba7ceb1ceb836286e12610a02.png)

**Comment by mdcfe** [comment]
Could you run `/ess debug` then try spawning the item in again?

**Comment by pop4959** [comment]
[https://hastebin.com/lodanakodi.md](https://hastebin.com/lodanakodi.md)

**Comment by mdcfe** [comment]
Thanks for reporting this - I accidentally mapped the vanilla `harming` potion to `PotionType.INSTANT_HARMING` (which doesn't exist) instead of `PotionType.INSTANT_DAMAGE`. This is fixed in 97cc3d00661664debfac791f06a6b94ee22c7907 and in the latest builds.
